mozilla/blurts-server
# Fix date input placeholder for iOS

# References: 
Jira: [MNTOR-1962](https://mozilla-hub.atlassian.net/browse/MNTOR-1962)



# Description

Fixes the visual appearance on iOS devices.

# Screenshot

**Before**
| Empty | Filled |
| --- | --- |
| ![dob-before-placeholder](https://github.com/mozilla/blurts-server/assets/13835474/f2d90457-ec8a-47f1-82be-639f576b446a) | ![dob-before-filled](https://github.com/mozilla/blurts-server/assets/13835474/8a9368f8-c098-4632-a78a-0c239798e29a) |

**After**
| Empty | Filled |
| --- | --- |
| ![dob-empty-after](https://github.com/mozilla/blurts-server/assets/13835474/b0106f2b-2347-47da-93fa-a12647173c9f) | ![dob-filled](https://github.com/mozilla/blurts-server/assets/13835474/56fa104a-0bb0-4615-82fb-7d5378f12c22) |


# How to test

- Visit https://deploy-preview-3468--fx-monitor-storybook.netlify.app/?path=/story/pages-onboarding--onboarding on an iOS device
- Make sure the “date of birth” input looks like on the screenshots above

[MNTOR-1962]: https://mozilla-hub.atlassian.net/browse/MNTOR-1962?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
